docs/source/deploypeer/peerdeploy.md

```diff
@@ -0,0 +1,111 @@
+
+**Tip:** Add the location of the peer binary to your `PATH` environment variable so that it can be picked up without fully qualifying the path to the binary executable, for example:
+
+```
+export PATH=<path to download location>/bin:$PATH
+```
+
+After you have mastered deploying and running a peer by using the peer binary and core.yaml configuration file, it is likely that you will want to use a peer container in a Kubernetes or Docker deployment. The Hyperledger Fabric project publishes a [peer image](https://hub.docker.com/r/hyperledger/fabric-peer) that can be used for development and test, and various vendors provide supported peer images. For now though, the purpose of this topic is to teach you how to properly use the peer binary so you can take that knowledge and apply it to the production environment of your choice.
+
+## Prerequisites
+
+Before you can launch a peer node in a production network, you need to configure it. Complete the following steps:
+
+1. Create the recommended folder structure for the peer certificates that is described in the [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) topic to store the generated certificates and MSPs. This folder structure isn't mandatory, but these instructions presume you have created it:
+    ```
+    ├── organizations
+      └── peerOrganizations
+          └── org1.example.com
+              ├── msp
+              └── peers
+                  └── peer0.org1.example.com
+                      ├── msp
+                      └── tls
+    ```
+2. While **cryptogen** is a convenient utility that can be used to generate certificates for a test network, it should **never** be used on a production network. The core requirement for certificates for Fabric nodes is that they are Elliptic Curve (EC) certificates. You can use any tool you prefer to issue these certificates (for example, OpenSSL). However, the Fabric CA streamlines the process because it generates the Membership Service Providers (MSPs) for you. Regardless of which certificate provider you choose, you should have already used it to generate the peer enrollment certificate, TLS certificate, private keys, and the MSPs that Fabric must consume. Refer to the [CA deployment](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/cadeploy.html) and [Registering and enrolling identities with a CA](https://hyperledger-fabric-ca.readthedocs.io/en/release-1.4/deployguide/use_CA.html) topics for instructions on how to generate these certificates. You need to generate the following sets of certificates:
+    - **Peer TLS certificates**
+    - **Peer organization admin certificates (organization MSP)**
```

Comment:
Note that this bullet makes it sound like we'll have an admin cert (or multiple certs). That's not the case when using admin NodeOUs. The NodeOU configuration indicates that ANY cert issued by the org CA with role of 'admin' should be considered an admin.
Actually I think that the bullet should be removed. The NodeOU is configured at the channel org MSP level, not the peer level... Unless I'm missing something...

Reply:
@pamandrejko I think Dave has a point here and made some edits here pointing to the registering and enrolling CA doc to make some edits here.

Reply:
Yeah. I was using "certificates" to mean certificate + private key. But I am fine changing it to certificate (singular) if that makes more sense here.